bootstrap.sh

```diff
@@ -130,16 +130,16 @@ fi
 # Produce the runtimes.
 if [ ! -e ${GEN_DIR}/ELF_32_linux.S ]; then
   echo "[+] Generating runtimes"
-  clang++-3.8 -m32 -std=gnu++11 ${DIR}/mcsema/Arch/X86/print_ELF_32_linux.cpp
+  clang++-3.8 -m32 -std=gnu++11 ${DIR}/mcsema/Arch/X86/Runtime/print_ELF_32_linux.cpp
   ./a.out > ${GEN_DIR}/ELF_32_linux.S
 
-  clang++-3.8 -std=gnu++11 ${DIR}/mcsema/Arch/X86/print_ELF_64_linux.cpp
+  clang++-3.8 -std=gnu++11 ${DIR}/mcsema/Arch/X86/Runtime/print_ELF_64_linux.cpp
```

Comment:
Also can we add explicit output file names via `-o`, so they're not all `a.out`?